src/main/java/edu/kis/powp/jobs2d/command/FigureCommandFabric.java

```diff
@@ -0,0 +1,38 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class FigureCommandFabric
+{
+    public static ComplexCommand DrawSquareCommand(Job2dDriver driver)
```

Comment:
DriverCommand return type would be sufficient.
The name should start from a lowercase